engine/flink/test-utils/src/main/scala/pl/touk/nussknacker/engine/flink/test/FlinkTestConfiguration.scala

```diff
@@ -12,17 +14,24 @@ object FlinkTestConfiguration {
     val config = new Configuration
     config.setInteger(ConfigConstants.LOCAL_NUMBER_TASK_MANAGER, taskManagersCount)
     config.setInteger(TaskManagerOptions.NUM_TASK_SLOTS, taskSlotsCount)
+
+    config.set(PipelineOptions.CLASSPATHS, classpathWorkaround.asJava)
+
+    setupMemory(config)
+  }
+
+  // FIXME: better describe which classpath is used in this case
```

Comment:
what about the fixme?

Reply:
I don't understand this hack, I didn't want to dig deeper into it. I have hope that someday someone explain what the original author did